static/css/legacy.less

```diff
@@ -1083,6 +1083,7 @@ div#searchResults ul {
 // openlibrary/templates/type/author/view.html
 // openlibrary/templates/work_search.html
 div#searchResults {
+  max-width: 100%;
```

Comment:
```suggestion
  max-width: 78%;

  @media screen and (max-width: @width-breakpoint-tablet) {
    max-width: 100%;
  }
```
It looks like 100% is too much:

![Screenshot from 2021-11-02 19-21-57](https://user-images.githubusercontent.com/28732543/139965283-c128ee5b-50b8-425b-bb9c-5797eae7d6d5.png)